RFS/src/test/java/com/rfs/integration/EndToEndTest.java

```diff
@@ -0,0 +1,65 @@
+package com.rfs.integration;
+
+import org.junit.jupiter.api.Test;
+import com.rfs.framework.SimpleRestoreFromSnapshot;
+
+/**
+ * Tests focused on setting up whole source clusters, performing a migration, and validation on the target cluster
+ */
+public class EndToEndTest {
+
+    public Object sourceCluster;
+    public Object targetCluster;
+    public SimpleRestoreFromSnapshot simpleRfsInstance;
+
+    @Test
+    public void ES_v6_8_to_OS_v2_X_Migration() throws Exception {
```

Comment:
Are there other 6.8 scenarios that we'd want end to end coverage on or is 1 enough?